package.json

```diff
@@ -66,5 +65,8 @@
 			"last 1 firefox version",
 			"last 1 safari version"
 		]
+	},
+	"devDependencies": {
+		"vite": "^4.1.4"
```

Comment:
Why move this one dependency specifically? Anyway, I have intentionally avoided use devDependencies to keep things simple.

Reply:
it auto did that in order to run the game locally